lib/checks/landmarks/landmark-is-unique.js

```diff
@@ -0,0 +1,7 @@
+let role = axe.commons.aria.getRole(node);
```

Comment:
is it worth adding `if(!role) return false`?

Reply:
The reason I didn't add it is because we can be certain by the time we get here, we know for sure that it has a role (since matches would filter out other scenarios).

If you still feel strongly that I should add it, I can.